taccsite_cms/static/site_cms/css/src/_imports/components/django.cms.blog.app.item.css

```
@@ -243,12 +243,9 @@ html:not(:--cms-edit-mode) .read-more a {
   @extend %x-article-link-text;
 }
 
-/* If editing, "read more" sits under article item nad has a note */
+/* If editing, hide "read more" */
+/* FAQ: User can: edit by double clicking abstract; visit by clicking title */
 html:--cms-edit-mode .read-more { grid-area: 3 / 1 / 4 / -1 }
 html:--cms-edit-mode .read-more::after {
```

This was not taking effect, because it was hiding a pseudo-element that did not exist, instead of hiding the actual element **and** it was still defining layout for that element we want to never see.

```suggestion
html:--cms-edit-mode .read-more {
```